typings/eslint-plugin-unicorn.d.ts

```diff
@@ -0,0 +1,15 @@
+declare module 'eslint-plugin-unicorn' {
```

Comment:
[Docs] For each of these, I think it'd be good to include a comment pointing to an issue on a repo asking for types. Their repo if they're in TypeScript; ours (to track adding to DefinitelyTyped) if not?